src/Components/SmartComponents/Dashbar/Dashbar.js

```diff
@@ -1,114 +1,99 @@
 import React from 'react';
-import { Card, Grid, GridItem, Split, SplitItem, StackItem, Stack, Alert, CardBody, Text } from '@patternfly/react-core';
+import { Card, Grid, GridItem, StackItem, Stack, Alert, CardBody, Text } from '@patternfly/react-core';
 import { Main } from '@redhat-cloud-services/frontend-components/Main';
 import { SecurityIcon } from '@patternfly/react-icons';
-import { CVES_ALLOWED_PARAMS } from '../../../Helpers/constants';
+import { CVES_ALLOWED_PARAMS, impactList, CVES_DEFAULT_FILTERS } from '../../../Helpers/constants';
 import { constructFilterParameters, useUrlParams } from '../../../Helpers/MiscHelper';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { changeCveListParameters } from '../../../Store/Actions/Actions';
 import { FormattedMessage } from 'react-intl';
 import messages from '../../../Messages';
+import propTypes from 'prop-types';
+import { buildActiveFilters, removeFilters } from '../../../Helpers/TableToolbarHelper';
+
+const DashbarItem = ({ title, count, impact, onLinkClick }) => {
+    return (
+        <Card isFullHeight className="card-box">
+            <CardBody>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold">
+                    {title}
+                </Text>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold pf-u-mt-xs">
+                    <a onClick={onLinkClick}>
+                        { Object.keys(impactList).includes(impact)
+                            ? <SecurityIcon
+                                color={impactList[impact].iconColor}
```

Comment:
We can remove `Object.keys(impactList).includes(impact)` and instead error check with `?.` in `color={impactList[impact]?.iconColor}`